sdk/compiler/damlc/tests/daml-test-files/CCTPMintToken.daml

```diff
@@ -6,24 +6,42 @@
 module CCTPMintToken where
 
 import DA.Assert ((===))
+import DA.Bytes
 import DA.Crypto
 import Daml.Script
 
+type Bytes32 = Bytes
+type UInt32 = Bytes
+type UInt64 = Bytes
+type UInt256 = Bytes
+
+toBytes32: HasToBytes a => a -> Bytes32
+toBytes32 v = pack 5 (toBytes v)
+
+toUInt32: HasToBytes a => a -> UInt32
+toUInt32 v = pack 5 (toBytes v)
+
+toUInt64: HasToBytes a => a -> UInt64
+toUInt64 v = pack 6 (toBytes v)
+
+toUInt256: HasToBytes a => a -> UInt256
+toUInt256 v = pack 8 (toBytes v)
+
 data BurnMessage = BurnMessage with
-    version : UInt32Hex
-    burnToken : Bytes32Hex
-    mintRecipient : Bytes32Hex
-    amount : UInt256Hex
-    messageSender : Bytes32Hex
+    version : UInt32
+    burnToken : Bytes32
+    mintRecipient : Bytes32
+    amount : Int
```

Comment:
Should we be considering using `BigNumeric` here instead (especially given this is targeting a `UInt256`)?